package.json

```diff
@@ -210,7 +210,7 @@
     ]
   },
   "engines": {
-    "node": "^10 || ^12"
+    "node": "^12 || ^14"
```

Comment:
@sneridagh @tiberiuichim this is a breaking change IMHO. So we either have to wait for the next major release or do a major release right away.

Reply:
Oddly enough, 10, 12 and 14 will be "valid" LTS until 2021-04-30 when they will drop 10 as LTS.
https://nodejs.org/en/about/releases/

We could maintain the node restriction, but remove it from the builds tests. Then we minimize the impact, and no need for a major... Doing a major is ok for me too.

Reply:
We have to come up with an official release policy here. I'd say we always support all LTS versions that are currently supported and drop them with a major release on the day they run out of support. We have to write this down somewhere in the docs. If we support a version we should test it.

It might also help to add docs how to override the pinned version in custom project. We had this problem in a project but I don't fully recall how I fixed it back then. :(

Reply:
Changing this should definitely be a breaking change. Always supporting all LTS versions is a safe policy, it requires a second PR in a few months though.
If we want to save that time, we could go for the "add this PR to the next major version", whenever that happens (I assume it will happen before April 30th). It's not that much time to be saved, actually, so I'm fine either way.

Reply:
We can always add a new LTS version. Which is something we should do as soon as it is out. With removing we should maybe wait for the next major version we do anyways.